docs/application/native/guides/app-management/dali-widget-app.md

```diff
@@ -0,0 +1,268 @@
+# DALi Widget Application
+
+
+You can create a widget applications with DALi to display small version of application on the homescreen.
+
+To create a DALi widget application, you must:
+
+- Use the [Dali::WidgetApplication class](#widget-application-api).
+
+	> **Note**
+	> A DALi widget application is implemented similarly to a [DALi basic UI application](dali-basic-app.md), since the [Dali::WidgetApplication](../../api/mobile/latest/classDali_1_1WidgetApplication.html) class inherits from the [Dali::Application](../../api/mobile/latest/classDali_1_1Application.html) class. Consequently, make sure you are familiar with the basic UI application details before tackling a widget application.
```

Comment:
Rephrase the note as:

- The implementation of DALi widget application is similar to DALi basic user interface application, because the  `Dali::WidgetApplication` class inherits from the `Dali::Application `class. 

- Ensure you are familiar with basic use interface application details before tracking a widget application.

Dali::Application> link not working
Dali::WidgetApplication > link not working